cli/src/command/account.rs

```diff
@@ -644,8 +664,19 @@ pub async fn node_info_command(account: &Account) -> Result<(), Error> {
 }
 
 /// `output` command
-pub async fn output_command(account: &Account, output_id: String) -> Result<(), Error> {
-    let output = account.get_output(&OutputId::from_str(&output_id)?).await;
+pub async fn output_command(account: &Account, selector: OutputSelector) -> Result<(), Error> {
+    let output = match selector {
+        OutputSelector::Id(id) => account.get_output(&id).await,
+        OutputSelector::Index(index) => {
+            let mut outputs = account.outputs(None).await?;
+            outputs.sort_by(|a, b| {
+                a.metadata
+                    .milestone_timestamp_booked()
+                    .cmp(&b.metadata.milestone_timestamp_booked())
+            });
+            outputs.into_iter().rev().nth(index)
```

Comment:
reversing won't help. Do you mean negating? I wouldn't negate `a`, `b` because then we'ld have to cast.

Reply:
Why wouldn't reversing help ?

Reply:
because (a, b, <=) or (b, a, <=) both produce the same result. Negating however (-a, -b, <=) and (-b, -a, <=) reverse the order.

Reply:
a, b are just variables not actual numbers

Reply:
wait a sec ... maybe I am having a brain fart

Reply:
Well yeah sure, but if you reverse the variables, it reverses the order 🤷🏻

Reply:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021

Reply:
I am an idiot ... somehow I thought `cmp(a, b)` would return the value of `a` if `a <= b`, and `b` otherwise. That would make the result independent of the order of the arguments. Forget about me ...